semantic/src/site/globals/site.overrides

```diff
@@ -857,6 +857,11 @@ a:hover {
   }
 }
 
+// when SummaryView elements used on Application pages
+.summary-extra-spacing {
```

Comment:
can we use .form-element with child selector for this?

.form-area {
...
  .summary-page-element-container {
    margin-bottom: 20px;
 }
}

maybe at the same time add variable for both element-application-view and this one for margin

Reply:
Yeah good point. This what you mean? 121b923b